TestCases/pastix_support/readme.txt

```diff
@@ -0,0 +1,40 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
@rsanfer, tutorial, so far the recipe has only been tested on Ubuntu 18.04 (gcc 7.4) OpenMPI 3.1.4, MKL 2017 or OpenBLAS 0.3.2

Reply:
Eventually we should put that to the docs on the website.